### DISCUSSION TITLE
Testers needed for bullseye version

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#227] [General]
BBN Marine OS is looking for volunteers to do more testing on its latest Bullseye based arm64 version for Raspberry Pi.

Please report any issues you find here:

https://github.com/bareboat-necessities/lysmarine_gen/issues

Fair Winds!

**Comment by Philip0123** [comment]
can I work with you as a volunteer?

**Reply by mgrouch** [reply]
Yes sure
